omod/src/main/java/org/openmrs/module/pihmalawi/rest/controller/YendaNafePatientRestController.java

```diff
@@ -0,0 +1,136 @@
+package org.openmrs.module.pihmalawi.rest.controller;
+
+import org.openmrs.api.LocationService;
+import org.openmrs.api.PatientService;
+import org.openmrs.api.PersonService;
+import org.openmrs.api.UserService;
+import org.openmrs.module.pihmalawi.models.YendaNafePatientRegistrationModel;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+@Controller
+@RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + YendaNafePatientRestController.PIHMALAWI + YendaNafePatientRestController.YENDANAFE+YendaNafePatientRestController.PATIENT)
+public class YendaNafePatientRestController {
+
+    public static final String PIHMALAWI = "/pihmalawi";
+    public static final String YENDANAFE = "/yendanafe";
+    public static final String PATIENT = "/patient";
+
+    public final String personUri = "http://localhost:8080/openmrs/ws/rest/v1/person";
+
+    @Autowired
+    private PatientService patientService;
+
+    @Autowired
+    private PersonService personService;
+
+    @Autowired
+    private UserService userService;
+
+    @Autowired
+    private LocationService locationService;
+    private Date date = new Date();
+
+    @RequestMapping(method = RequestMethod.POST)
+    @ResponseBody
+    public Object getScreeningData(
```

Comment:
Hi @kmatiya , this looks like just the skeleton of a method you copied from the other Rest controller? This is not meant to be the implementation of the YendaNafe registration?